src/solver/conv_asm_implicit_gemm_gtc_bwd.cpp

```diff
@@ -202,15 +197,16 @@ static bool FindImplicitGemmGtcDynamicBwdKernel(const ConvolutionContext& ctx,
                 continue;
             }
         }
-        if((gemm_n % pConfig->gemm_n_per_block != 0) || (gemm_m % pConfig->gemm_m_per_block != 0))
+        if((pConfig->gemm_n_per_block == 0) || (pConfig->gemm_m_per_block == 0) ||
+           (gemm_n % pConfig->gemm_n_per_block != 0) || (gemm_m % pConfig->gemm_m_per_block != 0))
         {
             continue;
         }
-        if(pConfig->gemm_n_per_block % pConfig->nxb != 0)
+        if((pConfig->nxb == 0) || (pConfig->gemm_n_per_block % pConfig->nxb != 0))
```

Comment:
Is it so that `nxb == 0` is normal situation?

Reply:
no, nxb means the factor of b(ho*wo), which should not be 0.

Reply:
But this case is handled like a normal one. Shall we handle this as an error, please.

Reply:
Yes, `nxb` or `gemm_k_per_block ` should never be `0`. This case should handle as error or assert, not in normal applicability check logic

Reply:
fix it by adding assert(false);